config/routes.rb

```diff
@@ -0,0 +1,23 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  resources :recipes do
+    resources :recipe_foods, as: 'foods'
+    # resources :shopinglists
```

Comment:
- [ ] @blase147 and @kaizipaul it's not good practice to have commented code in the code base kindly remove it